.snyk

```diff
@@ -1,6 +1,11 @@
 # Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
-version: v1.14.1
-ignore: {}
+version: v1.19.0
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
+ignore:
+  SNYK-JS-SSRI-1085630:
```

Comment:
This change is outdated. A remediation is now available. https://snyk.io/vuln/SNYK-JS-SSRI-1085630#remediation